scripts/format-queries.lua

```diff
@@ -0,0 +1,460 @@
+(program
+  (_) @format.append-newline)
+(program
+  (_) @format.cancel-append .)
+(program
+  . (_)
+  [
+    (list)
+    (grouping)
+    (named_node)
+    (anonymous_node)
+    (field_definition)
+  ] @format.prepend-newline)
+
+(program
+  (comment) @_comment
+  .
+  [
+    (list)
+    (grouping)
+    (named_node)
+    (anonymous_node)
+    (field_definition)
+    (comment)
+  ] @format.cancel-prepend
+  (#is-start-of-line? @_comment)
+  (#not-lua-match? @_comment "^;+%s*inherits:")
+  (#not-lua-match? @_comment "^;+%s*extends%s*$"))
+;; }}}
+
+;; delims
+[
+  ":"
+  "."
+] @format.append-space
+(
+  "." @format.prepend-space @format.cancel-append
+  .
+  ")")
+
+;; List handler
+;; Only starts indent if 2 or more elements
+(list
+  "[" @format.indent.begin
+  "]" @format.indent.dedent)
+;; Otherwise, remove brackets
+(list
+  "[" @format.remove @format.cancel-append
+  .
+  (_) @format.cancel-append
+  .
+  "]" @format.remove)
+;; [ ... ] @capture1 @capture2
+;; Append newlines for nodes inside the list
+(list
+  (_) @format.append-newline
+  (#not-has-type? @format.append-newline capture quantifier))
+
+;; (_), "_" and _ handler
+;; Start indents if it's one of these patterns
+(named_node
+  [
+    "_"
+    name: (identifier)
+  ] @format.indent.begin
+  .
+  [
+    (list)              ; (foo [...])
+    (grouping)          ; (foo ((foo)))
+    (negated_field)     ; (foo !field)
+    (field_definition)  ; (foo field: (...))
+    (named_node)        ; (foo (bar))
+    (predicate)         ; (named_node (#set!))
+    (anonymous_node)
+    "."
+  ])
+;; Honoring comment's position within a node
+(named_node
+  [
+    "_"
+    name: (identifier)
+  ] @format.indent.begin
+  .
+  (comment) @_comment
+  (#is-start-of-line? @_comment))
+(named_node
+  [
+    "_"
+    name: (identifier)
+  ] @format.indent.begin @format.cancel-append
+  .
+  "."? @format.prepend-newline
+  .
+  (comment) @format.prepend-space
+  (#not-is-start-of-line? @format.prepend-space))
+
+;; Add newlines for other nodes, in case the top node is indented
+(named_node
+  [
+    (list)
+    (grouping)
+    (negated_field)
+    (field_definition)
+    (named_node)
+    (predicate)
+    (anonymous_node)
+    "."
+  ] @format.append-newline)
+
+;; Collapse closing parentheses
+(named_node
+  [
+    "_"
+    name: (identifier)
+    (_)
+  ] @format.cancel-append
+  .
+  ")"
+  (#not-has-type? @format.cancel-append comment))
+
+;; All captures should be separated with a space
+(capture) @format.prepend-space
+
+;; Workaround to just use the string's content
+(anonymous_node (identifier) @format.keep)
+(field_definition
+  name: (_)
+  ":" @format.indent.begin @format.append-newline ; suppress trailing whitespaces with forced newlines
+  [
+    (named_node [ (named_node) (list) (grouping) (anonymous_node) (field_definition) ])
+    (list "[" . (_) . (_) "]")
+    (grouping "(" . (_) . (_) ")")
+    (grouping
+      quantifier: (quantifier))
+  ])
```

Comment:
No wonder :D. This has been removed on nvim-treesitter for a while now. It's best to do a carbon copy of latest nvim-treesitter of today, or to do what's in TheLeoP refactor branch of curl-ing the format-queries file

Reply:
Ah oops thanks! Thought I did the latest, but maybe I should have done main instead. I will try and use the curl method though, that seems like the cleanest

Reply:
Ok, I believe this is resolved. Thank you for the code @TheLeoP!